### ISSUE TITLE
Fatal error - app dont work

### ISSUE THREAD

**Issue by wanick** [StarlaneStudios/Surrealist#178]
**Describe the bug**
Open Surrealist app and see 

```
Unexpected Application Error!
unreachable
RuntimeError: unreachable
    at __rust_start_panic (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[29409]:0xf4e6ae)
    at rust_panic (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[28455]:0xf4b8ab)
    at std::panicking::rust_panic_with_hook::h1e6ac5d404b8e31b (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[12753]:0xda6a97)
    at std::panicking::begin_panic_handler::{{closure}}::h24b0f4622f2766a5 (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[15519]:0xe2f16f)
    at std::sys_common::backtrace::__rust_end_short_backtrace::h19f35d272c126e7c (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[29252]:0xf4e238)
    at rust_begin_unwind (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[23602]:0xf1619f)
    at core::panicking::panic_fmt::h87755523850ece9e (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[24608]:0xf27557)
    at core::str::slice_error_fail_rt::h91bee75a4bf13f5a (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[1905]:0x842bd4)
    at core::str::slice_error_fail::h37edf3bde95d9b91 (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[28008]:0xf498dc)
    at surrealdb_core::syn::common::Location::of_in::hcf02b4219298792b (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[9709]:0xcdaf92)
```
restar PC dont support, reinstall App dont support

F12 -> console 
```
query
index-28beb63e.js:117  panicked at /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/surrealdb-core-1.2.0/src/syn/common.rs:32:34:
byte index 12 is out of bounds of `FETCH user`

Stack:

Error
    at n.wbg.__wbg_new_abda76e883ba8a5f (http://localhost:19479/assets/index-28beb63e.js:117:6189)
    at console_error_panic_hook::hook::h8729094ed15ceab7 (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[8539]:0xc7bcca)
    at core::ops::function::Fn::call::h5768130c9a047ad5 (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[29257]:0xf4e269)
    at std::panicking::rust_panic_with_hook::h1e6ac5d404b8e31b (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[12753]:0xda6a67)
    at std::panicking::begin_panic_handler::{{closure}}::h24b0f4622f2766a5 (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[15519]:0xe2f16f)
    at std::sys_common::backtrace::__rust_end_short_backtrace::h19f35d272c126e7c (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[29252]:0xf4e238)
    at rust_begin_unwind (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[23602]:0xf1619f)
    at core::panicking::panic_fmt::h87755523850ece9e (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[24608]:0xf27557)
    at core::str::slice_error_fail_rt::h91bee75a4bf13f5a (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[1905]:0x842bd4)
    at core::str::slice_error_fail::h37edf3bde95d9b91 (http://localhost:19479/assets/surrealist-embed_bg-a3eeaea3.wasm:wasm-function[28008]:0xf498dc)


n.wbg.__wbg_error_f851667af71bcfc6 @ index-28beb63e.js:117
index-28beb63e.js:90  RuntimeError: unreachable
    at __rust_start_panic (surrealist-embed_bg-a3eeaea3.wasm:0xf4e6ae)
    at rust_panic (surrealist-embed_bg-a3eeaea3.wasm:0xf4b8ab)
    at std::panicking::rust_panic_with_hook::h1e6ac5d404b8e31b (surrealist-embed_bg-a3eeaea3.wasm:0xda6a97)
    at std::panicking::begin_panic_handler::{{closure}}::h24b0f4622f2766a5 (surrealist-embed_bg-a3eeaea3.wasm:0xe2f16f)
    at std::sys_common::backtrace::__rust_end_short_backtrace::h19f35d272c126e7c (surrealist-embed_bg-a3eeaea3.wasm:0xf4e238)
    at rust_begin_unwind (surrealist-embed_bg-a3eeaea3.wasm:0xf1619f)
    at core::panicking::panic_fmt::h87755523850ece9e (surrealist-embed_bg-a3eeaea3.wasm:0xf27557)
    at core::str::slice_error_fail_rt::h91bee75a4bf13f5a (surrealist-embed_bg-a3eeaea3.wasm:0x842bd4)
    at core::str::slice_error_fail::h37edf3bde95d9b91 (surrealist-embed_bg-a3eeaea3.wasm:0xf498dc)
    at surrealdb_core::syn::common::Location::of_in::hcf02b4219298792b (surrealist-embed_bg-a3eeaea3.wasm:0xcdaf92)
pq @ index-28beb63e.js:90
index-28beb63e.js:1247  React Router caught the following error during render RuntimeError: unreachable
    at __rust_start_panic (surrealist-embed_bg-a3eeaea3.wasm:0xf4e6ae)
    at rust_panic (surrealist-embed_bg-a3eeaea3.wasm:0xf4b8ab)
    at std::panicking::rust_panic_with_hook::h1e6ac5d404b8e31b (surrealist-embed_bg-a3eeaea3.wasm:0xda6a97)
    at std::panicking::begin_panic_handler::{{closure}}::h24b0f4622f2766a5 (surrealist-embed_bg-a3eeaea3.wasm:0xe2f16f)
    at std::sys_common::backtrace::__rust_end_short_backtrace::h19f35d272c126e7c (surrealist-embed_bg-a3eeaea3.wasm:0xf4e238)
    at rust_begin_unwind (surrealist-embed_bg-a3eeaea3.wasm:0xf1619f)
    at core::panicking::panic_fmt::h87755523850ece9e (surrealist-embed_bg-a3eeaea3.wasm:0xf27557)
    at core::str::slice_error_fail_rt::h91bee75a4bf13f5a (surrealist-embed_bg-a3eeaea3.wasm:0x842bd4)
    at core::str::slice_error_fail::h37edf3bde95d9b91 (surrealist-embed_bg-a3eeaea3.wasm:0xf498dc)
    at surrealdb_core::syn::common::Location::of_in::hcf02b4219298792b (surrealist-embed_bg-a3eeaea3.wasm:0xcdaf92) Object
componentDidCatch @ index-28beb63e.js:1247
index-28beb63e.js:117 Connecting to localhost:8000
index-28beb63e.js:117 Executing remote query INFO FOR KV
index-28beb63e.js:117 Executing remote query INFO FOR NS
index-28beb63e.js:117 Executing remote query INFO FOR DB
index-28beb63e.js:117 Querying database version
index-28beb63e.js:117 Database version is 1.2.1
3index-28beb63e.js:117 Received response from database, success: true
index-28beb63e.js:117 Executing remote query INFO FOR TABLE `queue`;INFO FOR TABLE `sessions`;INFO FOR TABLE `user`;
index-28beb63e.js:117 Received response from database, success: true
```

**Expected behavior**

**Environment (you can view the current version under Settings):**
 - Operating System [e.g. Windows]: Windows
 - Surrealist Version [e.g. 1.2.0]:  surrealist-v1.11.4-windows
 - SurrealDB Version [execute "surreal version"]: 1.2.1

**Comment by wanick** [comment]
It is ok,  

remove file   AppData\Roaming\surrealdb.json

**Comment by macjuul** [comment]
The actual cause of this issue has now been fixed in SurrealDB v1.2.1 ([releases](https://surrealdb.com/releases))
